libs/feature/map/src/lib/map-context/component/map-context.component.ts

```diff
@@ -41,25 +41,17 @@ export class MapContextComponent implements OnChanges {
 
   ngOnChanges() {
     if (this.context) {
-      if (this.context.extent) {
-        if (!this.map.getSize()) {
-          this.map.once('change:size', () => {
-            this.updateContextByExtent()
-          })
-        } else {
-          this.updateContextByExtent()
-        }
+      if (this.context.extent && !this.map.getSize()) {
```

Comment:
`extent` should go in the view

Reply:
I agree that the `view` appears fitting to include the `extent`. I was hesitating doing this, as it will mean to make `zoom` and `center` optional in our view model, which are mandatory to create an `ol.view`. I know, we will calculate them from the `extent` in time, but it seemed somehow misleading at first glance. Will see to adapt this.